src/Codec/Xlsx/Parser.hs

```diff
@@ -136,69 +156,65 @@ extractCellValue _ "b" "0" = [CellBool False]
 extractCellValue _ _ _ = []
 
 -- | Get xml cursor from the specified file inside the zip archive.
-xmlCursor :: Zip.Archive -> FilePath -> Maybe Cursor
-xmlCursor ar fname = parse <$> Zip.findEntryByPath fname ar
+xmlCursor :: Zip.Archive -> FilePath -> Parser (Maybe Cursor)
+xmlCursor ar fname = maybe (Right Nothing) parse $ Zip.findEntryByPath fname ar
```

Comment:
The problem I was having with this is that `xmlCursorRequired` can fail in two different ways: `MissingFile` or `InvalidFile`. We only want to catch `MissingFile` for the purpose of returning Nothing; if the file is present but invalid, the `InvalidFile` should still occur.